core/utilities/functions/_set-container-padding.scss

```diff
@@ -1,40 +1,35 @@
 @charset "UTF-8";
 
-// ========================================================
-// function set-container-padding()
-//
-// This function is called by the centered-section mixin.
-// It's important to note that the intended use of this,
-// along with the companion set-container-margin function
-// is to modify a container that occupies the entire width
-// of the browser, and not a nested container with it's own
-// margin.
-// ========================================================
+/// Set Container padding
+/// Creates a fluid container of a certain number of columns.
+// Can be constrained to a specific max width, or to the project's global `$max-container-width`.
 
-@function set-container-padding(
-  $_cols: null,
-  $_v-space: 1em,
-  $_grid: null,
-  $_max-width: false
+@function _set-container-padding(
+  $columns: null,
+  $grid: $neat-grid,
+  $v-space: 1em,
+  $max-width: false
 )
 {
-  $_gutter: _retrieve-neat-setting($_grid, gutter);
+  $columns: _neat-column-default($grid, $columns); // Retrieves max columns from specified grid if empty, or the specified number of columns otherwise
```

Comment:
Although in this instance, I would be ok using $_columns for this. At least it would be obvious that we're using a different value.